<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LXQtSensorsConfiguration</class>
 <widget class="QDialog" name="LXQtSensorsConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensors Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="mainTabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="commonTab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="updateIntervalL">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Update interval (seconds)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="updateIntervalSB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="tempBarWidthL">
         <property name="text">
          <string>Temperature bar width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="tempBarWidthSB">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="warningAboutHighTemperatureChB">
         <property name="toolTip">
          <string>Blink status bars when the temperature is too high</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Warning about high temperature</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="tempScaleGB">
         <property name="title">
          <string>Temperature scale</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="celsiusTempScaleRB">
            <property name="text">
             <string>Celsius</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fahrenheitTempScaleRB">
            <property name="text">
             <string>Fahrenheit</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>fahrenheitTempScaleRB</zorder>
         <zorder>celsiusTempScaleRB</zorder>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sensorsSettingsTab">
      <attribute name="title">
       <string>Sensors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="detectedChipsL">
         <property name="text">
          <string>Detected chips:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QComboBox" name="detectedChipsCB">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="chipFeaturesL">
         <property name="text">
          <string>Chip features:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QTableWidget" name="chipFeaturesT">
         <property name="columnCount">
          <number>3</number>
         </property>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>LXQtSensorsConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>LXQtSensorsConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
